patch-tracing-channel-has-subscribers.js

```diff
@@ -0,0 +1,44 @@
+const {
+  ReflectApply,
+  PromiseReject,
+  PromiseResolve,
+  PromisePrototypeThen,
+  ArrayPrototypeSplice,
+  ArrayPrototypeAt,
+} = require('./primordials.js');
+
+const { ERR_INVALID_ARG_TYPE } = require('./errors.js');
+
+const traceEvents = [
+  'start',
+  'end',
+  'asyncStart',
+  'asyncEnd',
+  'error',
+];
+
+module.exports = function (unpatched) {
+  const { channel } = unpatched;
+
+  const dc = { ...unpatched };
+
+  const tracingChannel = dc.tracingChannel;
+
+  dc.tracingChannel = function() {
+    const tc = tracingChannel(...arguments);
+
+    Object.defineProperty(tc, 'hasSubscribers', { // TODO: on prototype?
```

Comment:
Could you do it on the prototype by creating a throwaway TracingChannel and getting its prototype?

Reply:
That's just crazy enough to work